### DISCUSSION TITLE
Drawer или Sidebar

### DISCUSSION THREAD

**Discussion by ukqfhf** [VKCOM/VKUI#5912] [Ideas] [answered]
Не нашел в документации компонента `Drawer` (или `Sidebar`). Компонент базовый и используется в экосистеме VK. 
Есть планы по реализации?

Референсы:

- [Drawer из MUI](https://mui.com/material-ui/react-drawer)
- [Drawer из chakra](https://chakra-ui.com/docs/components/drawer)
- [Sidebar из primereact](https://primereact.org/sidebar/)

**Comment by BlackySoul** [comment]
Привет!  Спасибо за предложение по улучшению библиотеки, такого у нас в планах пока нет х)
Для реализации компонент должен удовлетворять [паре условий](https://github.com/VKCOM/VKUI/blob/master/docs/MANIFESTO.md#:~:text=%D0%9A%D0%BE%D0%BC%D0%BF%D0%BE%D0%BD%D0%B5%D0%BD%D1%82%20%D0%B2%20VKUI%20%D0%B4%D0%BE%D0%BB%D0%B6%D0%B5%D0%BD%20%D1%83%D0%B4%D0%BE%D0%B2%D0%BB%D0%B5%D1%82%D0%B2%D0%BE%D1%80%D1%8F%D1%82%D1%8C%20%D0%B4%D0%B2%D1%83%D0%BC%20%D0%BA%D1%80%D0%B8%D1%82%D0%B5%D1%80%D0%B8%D1%8F%D0%BC), видимо, это выполняется, но ещё мы исходим из концепции `design first`, поэтому обсудим это с нашими дизайнерами и вернемся с ответом 🙂

**Reply by ukqfhf** [reply]
Есть какой-то прогресс?

**Reply by BlackySoul** [reply]
Привет, пока нет (будет либо завтра, либо в ПН)

**Reply by BlackySoul** [reply, marked as answer]
Привет, дизайнеры не против добавить такой компонент R нам, пока находится у них на проработке
@Fliqle @Zaycevq
